### ISSUE TITLE
impossible to return to custom recovery 

### ISSUE THREAD

**Issue by leonfromotown** [M1cha/android_device_xiaomi_aries#78]
hey, 

I checked out the cm recovery for the fun of it, but now I can't install cwm or twrp which I used before. Neither can I flash another rom except for the ones m1cha provides. 

any suggestions before I use fastboot? :/

thanks in advance :)

**Comment by leonfromotown** [comment]
*in cm-12-20150302

**Comment by M1cha** [comment]
Just disable recovery flashing in developer settings :)

**Comment by leonfromotown** [comment]
I did, but cm recovery always tells me that either the signature is corrupt or some files could not be installed when I try to flash a new recovery :/

**Comment by leonfromotown** [comment]
die genaue fehlermeldung für cwm 6.0.3.1 ist:
script aborted: assert failed: getprop("ro.product.device") == "aries" | |
E: failed to install /sideload/package.zip

**Comment by M1cha** [comment]
mh that sounds weird indeed. That sounds like you would try to install TWRP for another device.

**Comment by leonfromotown** [comment]
well I tried different versions from different sources in order to rule out that very problem. also, I tried to flash cwm-packages that worked before. 
but by now, I returned to miui v6 with miflash bc I had some problems with the gapps/play services as well, so the thread can be closed although not solved. 
but props for the good work on a clean a5.0 for Aries :)
I enjoyed your rom 
On Apr 10, 2015 9:04 AM, Michael Zimmermann notifications@github.com wrote:mh that sounds weird indeed. That sounds like you would try to install TWRP for another device.
